Enable/Utilities/GoogleUtilities.h

```diff
@@ -7,7 +7,8 @@
 
 
 #import <GooglePlaces/GooglePlaces.h>
+#import <UIKit/UIKit.h>
 
 @interface GoogleUtilities : NSObject
-+ (void) getPlaceDataFromPOI_idStr:(NSString * _Nonnull)POI_idStr withFields: (GMSPlaceField) fields withCompletion: (void (^_Nonnull)(GMSPlace * _Nullable place)) completion;
++ (void) getPlaceDataFromPOI_idStr:(NSString * _Nonnull)POI_idStr withFields: (GMSPlaceField) fields withVC:(UIViewController* _Nonnull) vc withCompletion: (void (^_Nonnull)(GMSPlace * _Nullable place)) completion;
```

Comment:
I'm not a big fan of passing view controllers here, I think business logic classes should return errors, and view controllers who call them should handle them (and display them)